Lib/subprocess.py

```diff
@@ -756,19 +756,37 @@ class Popen:
       encoding and errors: Text mode encoding and error handling to use for
           file objects stdin, stdout and stderr.
 
+      read_callback: Callable to handle reading from the output and error pipes.
+          It takes a Popen object, a bool indictating whether the data came
+          from the child's stdout (True) or stderr (False), and the bytes
+          buffer to append the data to.
+
+      tee: If true, the output and/or error streams are also written to
+          the parent's sys.stdout and/or sys.stderr, respectively.
+
     Attributes:
         stdin, stdout, stderr, pid, returncode
     """
     _child_created = False  # Set here since __del__ checks it
 
+    def _read_handler(self, is_stdout, buffer, data):
```

Comment:
class organization style wise, never write a method before the constructor in the class body.

Reply:
Done.